pkg/keyctl/sys_linux.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 Jesse Sipprell. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
+// +build 386 amd64
```

Comment:
Actually, we need this at a few other places as well :( I'll drop comments at the places.